modules/RTC/JitsiTrack.js

```diff
@@ -310,4 +312,46 @@ JitsiTrack.prototype.getMSID = function () {
     return (streamId && trackId) ? (streamId + " " + trackId) : null;
 };
 
+/**
+ * Set new audio output device for track's DOM elements.
+ * @param {string} audioOutputDeviceId - id of 'audiooutput' device from
+ *      navigator.mediaDevices.enumerateDevices()
+ * @emits JitsiTrackEvents.TRACK_AUDIO_OUTPUT_CHANGED
+ * @returns {Promise}
+ */
+JitsiTrack.prototype.setAudioOutput = function (audioOutputDeviceId) {
+    var self = this;
+
+    if (!RTCUtils.isAudioOutputDeviceChangeAvailable()) {
+        return Promise.reject(
+            new Error('Audio output device change is not supported'));
+    }
+
+    return Promise.all(this.containers.map(function(element) {
+        return element.setSinkId(audioOutputDeviceId)
+            .catch(function (error) {
+                console.error('Failed to change audio output device on element',
+                    element, error);
+
+                // TODO: for some reason 'AbortError' is raised on video
```

Comment:
Actually found the reason why this happens!
The error happens when we assign MediaStream from webcam to video element with
```
element.src = URL.createObjectURL(stream);
```
if we do this through srcObject, everything works OK:
```
element.srcObject = stream;
```
Going to fix this next week.